resources/assets/coffee/react/profile-page/header-info.coffee

```diff
@@ -32,7 +32,9 @@ ProfilePage.HeaderInfo = ({user}) ->
       # hard space if no title
       span className: 'profile-info__title', user.title ? '\u00A0'
       div className: 'profile-info__flags',
-        el FlagCountry, country: user.country
+        a
+          href: laroute.route('rankings', {mode: currentMode, country: user.country.code, type: 'performance'})
```

Comment:
By the way, you can do it like this:

```coffee
a
  href: laroute.route 'rankings',
    mode: currentMode,
    country: user.country.code,
    type: 'performance'
```

It's more coffeescripty (?) like this.